cookbooks/cdo-varnish/libraries/http_cache.rb

```diff
@@ -76,6 +76,9 @@ def self.config(env)
               /
               /learn*
               /congrats
+              /mc
+              /starwars
+              /playlab
```

Comment:
The failing CircleCI test is warning that these additional configurations would put our CloudFront distribution config past our current limit of 25 cache behaviors per distribution. Unless we can find a way to consolidate some of these routes into a common subpath (not very likely considering the time remaining before HoC) I'll need to open an AWS support request and update this PR when the limit has been increased.